sriov/common/utils.py

```diff
@@ -18,6 +18,7 @@ def get_pci_address(ssh_obj: ShellHandler, iface: str) -> str:
         Exception: command failure
     """
     cmd = "ethtool -i {}".format(iface) + " | awk '/bus-info:/{print $2;}'"
+    print(ssh_obj.name + ": " + cmd)
```

Comment:
@dkosteck it would be better if we make a common function for this, for example as  a ShellHandler method. So users can  do ssh_obj.print(). This way if we decide later we want to change the print format, we just need to update a single function. 

Right now I feel we only need to add "trafficgen: " prefix for trafficgen, no need to add such prefix for dut, this way people can do copy and paste in block for dut.

Reply:
@dkosteck A second thought, maybe make a util print function?